Chat/FileManager.cpp

```diff
@@ -39,6 +39,16 @@ std::string FileManager::GetFileName(int file_index)const
     return shared_files_[file_index].GetName();
 }
 
+const double FileManager::GetFileSizeKB(int file_index)const
+{
+    return shared_files_[file_index].GetSizeKB();
```

Comment:
Please, add checking for array out of range for index to avoid crashes here and in lines 39 and 49